src/InstagramClone.Api/Startup.cs

```diff
@@ -25,7 +26,7 @@ private static void ConfigureService(WebApplicationBuilder builder)
                 config.ReturnHttpNotAcceptable = true;
             }).AddXmlDataContractSerializerFormatters();
 
-           // builder.Services.AddControllers();
+             // builder.Services.AddControllers();
```

Comment:
why did you comment out the controllers configuration? how will asp.net see and register your controllers?

Reply:
ooh, I missed it up.